### ISSUE TITLE
[CTD][1.16.4] Crash when creating new world with 11.1.8

### ISSUE THREAD

**Issue by drakray** [MinecraftModDevelopment/Extra-Golems#61]
https://pastebin.com/17ssEYus

```
Description: Exception in server tick loop

java.lang.ClassCastException: java.lang.Float cannot be cast to java.lang.Double
	at com.mcmoddev.golems.entity.base.GolemBase.getConfigDouble(GolemBase.java:254) ~[golems:11.1.8] {re:classloading}
	at com.mcmoddev.golems.entity.modded.UraniumGolem.<init>(UraniumGolem.java:37) ~[golems:11.1.8] {re:classloading}
	at net.minecraft.entity.EntityType.func_200721_a(EntityType.java:445) ~[?:?] {re:classloading,re:mixin}
	at com.natamus.collective.functions.ItemFunctions.generateEntityDropsFromLootTable(ItemFunctions.java:56) ~[collective:1.53] {re:classloading}
	at com.natamus.allloot.events.EntityEvent.onWorldLoad(EntityEvent.java:37) ~[alllootdrops:1.0] {re:classloading}
	at net.minecraftforge.eventbus.ASMEventHandler_1147_EntityEvent_onWorldLoad_Load.invoke(.dynamic) ~[?:?] {}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85) ~[eventbus-3.0.5-service.jar:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:297) ~[eventbus-3.0.5-service.jar:?] {}
	at net.minecraft.server.MinecraftServer.func_240787_a_(MinecraftServer.java:379) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:structure_gel.mixins.json:MinecraftServerMixin,pl:mixin:APP:charm.mixins.json:accessor.MinecraftServerAccessor,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.func_240800_l__(MinecraftServer.java:308) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:structure_gel.mixins.json:MinecraftServerMixin,pl:mixin:APP:charm.mixins.json:accessor.MinecraftServerAccessor,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:A}
	at net.minecraft.server.integrated.IntegratedServer.func_71197_b(IntegratedServer.java:60) ~[?:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.server.MinecraftServer.func_240802_v_(MinecraftServer.java:620) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:structure_gel.mixins.json:MinecraftServerMixin,pl:mixin:APP:charm.mixins.json:accessor.MinecraftServerAccessor,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:A}
	at net.minecraft.server.MinecraftServer.lambda$startServer$0(MinecraftServer.java:232) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:structure_gel.mixins.json:MinecraftServerMixin,pl:mixin:APP:charm.mixins.json:accessor.MinecraftServerAccessor,pl:mixin:APP:byg.mixins.json:server.MixinMinecraftServer,pl:mixin:A}
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_251] {}
```

MC 1.16.4
Forge 35.0.18
ExtraGolem 11.1.8
modList in log

**Comment by skyjay1** [comment]
Thanks for the report, I've uploaded a fix in 11.1.9 
I only had to change one single letter that turned a double into a float >_<

**Comment by drakray** [comment]
Yeah, that, and putting a Colon instead of a Semicolon, and everything catch fire(or just doesn't compile at all) XDXD
